rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -25,7 +25,7 @@
 import ctypes
 
 class ROCALGenericIterator(object):
-    def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = False, multiplier = [1.0,1.0,1.0], offset = [0.0, 0.0, 0.0], tensor_dtype = types.FLOAT, device = "cpu", device_id = 0):
+    def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = False, multiplier = [1.0,1.0,1.0], offset = [0.0, 0.0, 0.0], tensor_dtype = types.FLOAT, device = "cpu", device_id = 0, display = False):
```

Comment:
The arguments do not have a space between = right ?
Lets change this

Reply:
Or we can issue another PR consolidating all the reformatting changes

Reply:
This has been changed in the readers PR where we do the changes for the iterators but I'll do this here also